src/controllers/movie.js

```diff
@@ -0,0 +1,94 @@
+const { Prisma } = require("@prisma/client")
+const prisma = require('../utils/prisma')
+
+// Get all movies
+const getMovies = async (req, res) => {
+    try {
+        const allMovies = await prisma.movie.findMany({
+            include: {
+                screenings: true,
+            }
+        })
+        res.status(200).json({
+            movies: allMovies
+        })
+    } catch (err) {
+        res.status(404).json({  
+            error: err
+         })
+    }
+}
+
+// Create a movie
+const createMovie = async (req, res) => {
+
+    //const { title, runtimeMins } = req.body
+
+    //console.log("hi", req.body)
```

Comment:
Commented code is usually considered "dead code" and usually should be removed before a PR gets merged into the main branch. The git history exists so you can go back and find old code